mux.c

```diff
@@ -71,7 +71,6 @@
 /* from ssh.c */
 extern int tty_flag;
 extern Options options;
-extern int stdin_null_flag;
 extern char *host;
 extern int subsystem_flag;
```

Comment:
Should `extern int subsystem_flag` be removed too?

Reply:
Of course! Fixed.